src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientGameOverSystem.java

```diff
@@ -86,20 +87,38 @@ public void onGameOver(GameOverEvent event, EntityRef entity) {
     }
 
     private void addPlayerStatisticsInfo(DeathScreen deathScreen) {
-        MigLayout migLayout = deathScreen.find("playerStatistics", MigLayout.class);
-        if (migLayout != null) {
+        MigLayout spadesTeamMigLayout = deathScreen.find("spadesTeamPlayerStatistics", MigLayout.class);
+        MigLayout heartsTeamMigLayout = deathScreen.find("heartsTeamPlayerStatistics", MigLayout.class);
+        if (spadesTeamMigLayout != null && heartsTeamMigLayout != null) {
             Iterable<EntityRef> characters = entityManager.getEntitiesWith(PlayerCharacterComponent.class, LASTeamComponent.class);
             for (EntityRef character : characters) {
                 EntityRef client = character.getOwner();
                 ClientComponent clientComponent = client.getComponent(ClientComponent.class);
-                migLayout.addWidget(new UILabel(PlayerUtil.getColoredPlayerName(clientComponent.clientInfo)), new MigLayout.CCHint());
+                String playerTeam = localPlayer.getCharacterEntity().getComponent(LASTeamComponent.class).team;
```

Comment:
Not sure how `localPlayer.getCharacterEntity()` works on multiplayer.

Reply:
Have you experimented with it in the different cases? The area can indeed be a bit tricky, between single player, headless servers, hosting clients, and purely joining clients

With `ClientGameOverSystem` being a `CLIENT` system you should be fine vs headless servers. The `localPlayer` should be _something_ in all three other cases, but it is hard to remember any subtle differences

Reply:
> The localPlayer should be something in all three other cases
Do you mean it might be something different in each loop?

Reply:
Sorry, just meant that in each way you can run the game the setup would differ. Per client. Not anything related to a loop. There should only ever be one `localPlayer` but potentially multiple _actual_ players :-)

I was able to test around a bit, I crashed headless but suspect that was from running multiple clients from one workspace in funny ways 😅